Frontend/CO.CDP.OrganisationApp.Tests/Pages/ShareInformation/ShareCodesListViewTests.cs

```diff
@@ -130,20 +131,6 @@ public async Task OnGetDownload_ShouldRedirectToPageNotFound_WhenShareCodeIsEmpt
         result.Should().BeOfType<RedirectResult>().Which.Url.Should().Be("/page-not-found");
     }
 
-    [Fact]
-    public async Task OnGetDownload_ShouldRedirectToPageNotFound_WhenFileIsNotFound()
-    {
-        var shareCode = "HDJ2123F";
-
-        _dataSharingApiClientMock
-            .Setup(x => x.GetSharedDataFileAsync(shareCode))
-            .ReturnsAsync((WebApiClient.FileResponse?)null);
-
-        var result = await _pageModel.OnGetDownload(shareCode);
-
-        result.Should().BeOfType<RedirectResult>().Which.Url.Should().Be("/page-not-found");
-    }
-
     [Fact]
```

Comment:
As above, shouldn't this test still be here, but with a ThrowsAsync instead of returning null?